src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -289,6 +276,17 @@ std::string MessageHelper::GetDeviceMacAddressForHandle(
   return device_mac_address;
 }
 
+std::string MessageHelper::CommonLanguageToString(
+    hmi_apis::Common_Language::eType language) {
+  using namespace NsSmartDeviceLink::NsSmartObjects;
+  const char* str = 0;
+  if (EnumConversionHelper<hmi_apis::Common_Language::eType>::EnumToCString(
+          language, &str)) {
```

Comment:
@LuxoftAKutsan ,  `EnumConversionHelper` contains container `std::map` that allocates and frees memory